packages/e2e-tests/specs/experiments/document-settings.test.js

```diff
@@ -13,6 +13,24 @@ import { addQueryArgs } from '@wordpress/url';
  */
 import { navigationPanel } from '../../experimental-features';
 
+async function getDocumentSettingsTitle() {
+	await page.waitForSelector( '.edit-site-document-actions__title' );
+
+	return page.$eval(
+		'.edit-site-document-actions__title',
+		( el ) => el.innerText
+	);
```

Comment:
To make it a little more succinct, I think you can do something like:
```js
const titleElement = await page.waitForSelector( '.edit-site-document-actions__title' );
return titleElement.evaluate( ( el ) => el.innerText );
```

Reply:
Thanks for the tip! I'll add it now